### PR TITLE
Add CLIENT INFO command as Redis

### PR DESCRIPTION
**Pull request by enjoy-binbin**
This PR add CLIENT INFO command as Redis, this command
returns information about the current client connection,
the output format is the same as CLIENT LIST command.

CLIENT INFO was added in Redis 6.2.0

### CODE REVIEW

File: tests/gocase/unit/introspection/introspection_test.go [hunk 1]
```diff
@@ -45,6 +45,11 @@ func TestIntrospection(t *testing.T) {
 		require.Regexp(t, "id=.* addr=.*:.* fd=.* name=.* age=.* idle=.* flags=N namespace=.* qbuf=.* .*obuf=.* cmd=client.*", v)
 	})
 
+	t.Run("CLIENT INFO", func(t *testing.T) {
+		v := rdb.Do(ctx, "CLIENT", "INFO").Val()
```
Comment (enjoy-binbin) [inline tests/gocase/unit/introspection/introspection_test.go:49]: note go-redis does not support CLIENT INFO yet (a pending PR, should be supported soon)
so here i am using Do

**Comment by git-hulk** [comment]
Cool, Thanks for your contribution. @enjoy-binbin We can use the command `./x.py format` to auto-format and make the linter happy.

**Review by torwig — COMMENTED** [review]
LGTM. @enjoy-binbin just run the command @git-hulk provided and commit changes.

**Review by git-hulk — APPROVED** [review]
LGTM

**Review by torwig — APPROVED** [review]

**Comment by git-hulk** [comment]
Thanks all, merging..

**Pull request merged**